### DISCUSSION TITLE
Mac Classic 1 7.5.5 Issue

### DISCUSSION THREAD

**Discussion by Scarcey** [erichelgeson/BlueSCSI-v1#52] [Q&A]
Hi all 
I finally got my hands on a bluescsi in AUS and am running into an issue. 
Have it pre installed with system 7.5.5 on a 16gb sd. the mac isnt detecting it. 

i have been told that 7.5 wont run on a partition this large, and that i need to make it 2gb. How do i go at accomplishing this? The Mac has 2mb of ram also (two sticks). 
Heres the current log file settings:

```
BlueSCSI <-> SD - https://github.com/erichelgeson/BlueSCSI
VERSION: 1.0-20210410
DEBUG:0 SCSI_SELECT:0 SDFAT_FILE_TYPE:3
SdFat version: 2.0.6
SdFat Max FileName Length: 32
Initialized SD Card - lets go!
Imagefile: HD00_512 System 7.5 4GB.hda / 4293918720bytes / 4193280KiB / 4095MiB
Not an image: ._HD00_512 System 7.5 4GB.hda
Not an image: LOG.txt
Not an image: ._LOG.txt
ID:LUN0:LUN1:
 0: 512:----:
 1:----:----:
 2:----:----:
 3:----:----:
 4:----:----:
 5:----:----:
 6:----:----:
Finished initialization of SCSI Devices - Entering main loop.
```

Thank you for your help. Another note, how can i add another sector/scsi id for file transfers, i would like to add games and software onto the HDD when it works (fingers crossed)

**Comment by erichelgeson** [comment]
Hi, where did you get the 4gb image from? Maybe try one of the other premade images on the usage page.

Please see the Usage page as well on how to add more drives - It's as easy as adding a file with the correct name.

https://github.com/erichelgeson/BlueSCSI/wiki/Usage
